core-api/src/main/java/com/optimizely/ab/Optimizely.java

```diff
@@ -104,7 +109,8 @@ private Optimizely(@Nonnull EventHandler eventHandler,
                        @Nullable UserProfileService userProfileService,
                        @Nonnull ProjectConfigManager projectConfigManager,
                        @Nullable OptimizelyConfigManager optimizelyConfigManager,
-                       @Nonnull NotificationCenter notificationCenter
+                       @Nonnull NotificationCenter notificationCenter,
+                       @Nonnull List<OptimizelyDecideOption> defaultDecideOptions
```

Comment:
can you add an overloaded method and pass `[]`  without defaultDecideOptions?

Reply:
@msohailhussain can you elaborate? this is a private constructor any defaults should be handled in the builder. I don't think we should add another constructor.

Reply:
Yeah i missed private, this is fine.